eng/pipelines/templates/stages/vmr-build.yml

```diff
@@ -182,4 +194,6 @@ stages:
         excludeSdkContentTests: true       # ✅
         overrideDistroDisablingSha1: false # 🚫
         runOnline: false                   # 🚫
+        prepNoArtifacts: true              # ✅
+        prepNoSdk: true                    # ✅
         reuseBuildArtifactsFrom: Fedora36_Offline
```

Comment:
This is fine for preview 2 but when backporting to main, I don't see these parameters as being necessary and add value.  In order words, you should only have to specify one parameter to enable a bootstrap leg instead of multiple. 
 We can use the existing state used to determine what prep is necessary (https://github.com/dotnet/installer/blob/main/eng/pipelines/templates/jobs/vmr-build.yml#L174).

Reply:
@MichaelSimons - You're suggesting that the `withPreviousSDK` state be used to determine this scenario? But `withPreviousSDK` is false for the `Fedora36_Offline_MsftSdk` build leg, which is the scenario where we'd want to enable this scenario.

Reply:
I am suggesting if either withPreviousSDK or reuseBuildArtifactsFrom is being used, then we would run prep to acquire only the prebuilts.